app/components/DepositWithdraw/WithdrawModalNew.jsx

```diff
@@ -0,0 +1,834 @@
+
+        balances.forEach((item)=>{
+          let id = item.get("asset_type");
+          let asset = assets.get(id);
+
+          if(asset && item.get("balance") > 0){
+            symbolsToInclude.push(asset.symbol);
+          }
+        });
+
+        let { halfWidth, leftColumn, rightColumn, buttonStyle } = this._getStyleHelpers();
+        let { onFocus, onBlur } = this._getBindingHelpers();
+
+        const shouldDisable = isBTS ? !quantity || !btsAccount : !assetAndGateway || !quantity || !address || !canCoverWithdrawal || addressError;
+        let storedAddresses = WithdrawAddresses.get(selectedAsset.toLowerCase());
+
+        return <div id="withdraw_modal_new">
+          <div className="Modal__header" style={{textAlign: "center"}}>
+              <img src={logo} /><br />
+              <p><Translate content="modal.withdraw.header" /></p>
+          </div>
+
+          {/*ASSET SELECTION*/}
+          <div style={{marginBottom: "1em"}}>
+            <DepositWithdrawAssetSelector onSelect={this.onAssetSelected.bind(this)} onChange={this.onAssetChanged.bind(this)} include={symbolsToInclude} selectOnBlur />
+          </div>
+
+          {
+            !isBTS && selectedAsset && !selectedGateway ?
+            <Translate content="modal.withdraw.no_gateways" /> : 
+            null
+          }
+
+          {/*GATEWAY SELECTION*/}
+          <div style={{marginBottom: "1em"}}>
+            { selectedGateway ? gatewaySelector.call(this, {
+                selectedGateway, 
+                gatewayStatus, 
+                nAvailableGateways, 
+                error: false,
+                onGatewayChanged: this.onGatewayChanged.bind(this)
+            }) : null}
+          </div>
+
+          {/*QUANTITY*/}
+          {
+            assetAndGateway || isBTS ? 
+            <div>
+              {(preferredCurrency) ? <div style={{fontSize: "0.8em", position: "absolute", right: "1.25em"}}>
+                <Translate content="modal.withdraw.available" />
+                <span style={{color: canCoverWithdrawal ? null : "red", cursor: "pointer", textDecoration: "underline"}} onClick={this.onClickAvailableBalance.bind(this, convertedBalance)}>
+                    {/*Some currencies do not appear in balances, display zero balance if not found*/}
+                    {withdrawalCurrencyBalanceId ? <BalanceComponent balance={withdrawalCurrencyBalanceId} /> : "0.00"}
+                </span>
+              </div> : null}
+              <label className="left-label">
+                <Translate content="modal.withdraw.quantity" />
+              </label>
+              <ExchangeInput value={quantity} onChange={this.onQuantityChanged.bind(this)} onFocus={onFocus} onBlur={onBlur} />
+            </div> : 
+            null
+          }
+
+          {
+            (assetAndGateway || isBTS) && !canCoverWithdrawal ? 
+            <div style={{marginBottom: "1em"}}>
+              <Translate content="modal.withdraw.cannot_cover" />
```

Comment:
`missing translation: en.modal.withdraw.cannot_cover`

Reply:
Must have gotten lost in the merge